app/assets/stylesheets/components/_save-this-page.scss

```
@@ -0,0 +1,5 @@
+.app-c-save-this-page {
+  border: 1px solid govuk-colour('mid-grey');
+  border-top: 2px solid govuk-colour('blue');
+  padding: govuk-spacing(4) govuk-spacing(4) 0 govuk-spacing(4);
```

Being really nit-picky here, you could shorthand this to...
```suggestion
  padding: govuk-spacing(4) govuk-spacing(4) 0;
```

---

Yep you're right, I missed that. Corrected ✅